packages/amplication-client/src/User/Login.tsx

```diff
@@ -87,12 +86,8 @@ const Login = () => {
       <span className={`${CLASS_NAME}__title`}>Hi There</span>
       <Formik initialValues={INITIAL_VALUES} onSubmit={handleSubmit}>
         <Form childrenAsBlocks>
-          {urlError && (
-            <div className={`${CLASS_NAME}__login-error`}>
-              <Icon icon="alert_circle" />
-              {urlError}
-            </div>
-          )}
+
+          <LoginErrorMessage error={urlError} className={CLASS_NAME}/>
```

Comment:
I think it is better to add the condition on the page instead of trusing the component to use a condition. The inner condition will add the outer div for no reason